deployers/servicedeployer.go

```diff
@@ -397,8 +407,55 @@ func (deployer *ServiceDeployer) createAction(pkgname string, action *whisk.Acti
 	if err != nil {
 		wskErr := err.(*whisk.WskError)
 		log.Printf("Got error creating action with error message: %v and error code: %v.\n", wskErr.Error(), wskErr.ExitCode)
+	} else {
+		// create the API gateway according to the action definition in the manifest.yaml
+		// if no errors happened during the action creation.
+		// since the spec for this is not publish, so comment below lines.
+
+		//_, _, err := deployer.Client.Apis.Insert(createApiEntity(deployer, action), true)
+		//if err != nil {
+		//	wskErr := err.(*whisk.WskError)
+		//	fmt.Printf("Got error creating API with error message: %v and error code: %v.\n", wskErr.Error(), wskErr.ExitCode)
+		//}
+
```

Comment:
Perhaps not a placeholder, since if we need support the api gateway full grammer, I'll remove it.